charts/dex/templates/ingress.yaml

```diff
@@ -18,11 +18,6 @@ metadata:
   name: {{ $fullName }}
   labels:
     {{- include "dex.labels" . | nindent 4 }}
-  {{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
```

Comment:
I think I like this change. We will control the annotations block with only the `annotations` value of the chart if I get it right. It looks like the separation of concerns.

Reply:
Nope, this just removes the duplication. The reason why this was added is to provide a forward compatible option for setting the className on older ingress versions without the user having to check that first. The annotation and the className are mutually exclusive (validation fails if both are provided).

Reply:
Make sense, thanks.